Packs/MicrosoftGraphMail/ReleaseNotes/1_1_20.md

```diff
@@ -0,0 +1,5 @@
+
+#### Integrations
+##### O365 Outlook Mail (Using Graph API)
+- Updated the Docker image to: *demisto/crypto:1.0.0.31577*.
+- Added the *use_full_email_body* parameter to allow fetching incidents with the entire email body.
```

Comment:
switch "fetch" to "display"
We actually are still fetching the whole body, just not displaying it when its a preview.